### ISSUE TITLE
Remove brand bar from www.humanitiesiowa.org

### ISSUE THREAD

**Issue by mjoneill** [uiowa/uiowa#2941]
As a possible funding agency visiting www.humanitiesiowa.org, I have an experience that does not indicate this is a University of Iowa property.

---

Tasks: 
- [x] Remove brand bar
- [x] Recreate a search experience using Google CSE

Noted: This doesn't appear to be a UI org, so not particularly concerned with branding on this.

5 points

**Comment by joewhitsitt** [comment]
Marking as blocked. It was my understanding that contact with the site owners had to happen first.

**Comment by mjoneill** [comment]
> Marking as blocked. It was my understanding that contact with the site owners had to happen first.

I just got clarification on Site Owner.  Pete forwarded a convo from Heather Plucar that suggests we can reach out to her.

**Comment by briand44** [comment]
I reached out to Heather and got approval to move forward on this. The scope is to remove the branding bar but keep site search functionality. Not urgent, can be planned for sprint 52.

**Comment by mjoneill** [comment]
> I reached out to Heather and got approval to move forward on this. The scope is to remove the branding bar but keep site search functionality. Not urgent, can be planned for sprint 52.

Sounds good.  Let's pull it to the top of +1 in case we start raiding the backlog and are looking for new work.  Otherwise, I think we have a plan!

TY

**Comment by GaryRidgway** [comment]
Okay, there has been a lot to unpack with this issue. First, upon pulling it down after some lengthy D7 setup, i learned that this is a Standard site, not a custom. this means that it is very hard to decouple the uiowa bar from other modules that require it. 

From there, we looked at other Search options and wanted to see how we were implementing those, and it seems like they are made with custom modules, which we were hoping to avoid. Additionally, it seems like the search on humanities is not working at all, displaying no results with any search. We are hoping to get a new search ID for the site, and we are hoping that fixes that problem.

Also, the search page has text above the search results displaying "Search the University of Iowa for TERMS". This should be removed to disassociate  UIowa Branding.

With all of this, we think the issue might need to be considered for repointing

**Comment by GaryRidgway** [comment]
from michael: 

It appears we have a problem with humanitiesiowa.com

Content exists at both the following URLS:
* https://www.humanitiesiowa.org/council-conducted-partnerships-and-public-programs
* https://humanitiesiowa.org/council-conducted-partnerships-and-public-programs

Search console account exists for non-www site

Search results (in google) exist for www version

I suspect CSE was set up for non-www version

**Comment by GaryRidgway** [comment]
An Issue that has popped up for us is that there are now ads on the humanities site search. Further investigation is needed to look in to the possibility of removal for these ads.

Sadly, I cannot see these ads on any browser, with or without adblock, so my capacity to assist with this is minimal. For that reason I am going to go ahead and un-assign myself.

**Comment by richardbporter** [comment]
I got this to work by changing the config back to use the UI Search CSE ID and updating the UI Search CSE to add the www.humanitiesiowa.org domain. This is what other vanity domains are doing including neuronext.org and iowamade.org. I don't think those custom CSE are in use and can probably be deleted. I'm guessing they were setup initially but never cleaned up. I deleted the humanitiesiowa.org CSE.
